### ISSUE TITLE
Safari Version 8.0.6 (10600.6.3) showBanner is not executed

### ISSUE THREAD

**Issue by merqurio** [Alex-D/Cookies-EU-banner#11]
Hi !

I have a problem on Safari Version 8.0.6 (10600.6.3), where the showBanner function is not executed.

I think the problem is that init() is not run and therefor showBanner neither. My guess is that the problem is in line [32](https://github.com/Alex-D/Cookies-EU-banner/blob/master/src/cookies-eu-banner.js#L32) when checkijng if it's a bot or not :
```javascript
> var a = new CookiesEuBanner(function(){console.log("Hello world")});
< undefined
> a.init()
< false
```

I wish this helps :grinning:

**Comment by Alex-D** [comment]
Hi!

I fact, there is a bug here. I've test and it's from: https://github.com/Alex-D/Cookies-EU-banner/blob/master/src/cookies-eu-banner.js#L107

It's works if I replace `!==` by just `!=`

I apply this patch ASAP.

**Comment by merqurio** [comment]
Wow , that was a fast answer, Thank you very much !
If there's something else I can help with, please let me know.

**Comment by Alex-D** [comment]
Fix'd in 1.2.7 by this commit: https://github.com/Alex-D/Cookies-EU-banner/commit/19c8b88e98351bf177c2c3736589fb5480406d77
